accord-core/src/main/java/accord/impl/AbstractConfigurationService.java

```diff
@@ -269,6 +269,13 @@ public synchronized void reportTopology(Topology topology, boolean startSync)
         }
 
         long lastAcked = epochs.lastAcknowledged;
+        // TODO (now, review): lastAcked == 0, lastReceived = 2
+        // if we wait for epoch=1.acknowledge the test seems to wait forever... looks like burn test doesn't ack epoch=1
+        if (lastAcked == 0 && lastReceived > 0)
```

Comment:
have to think about this.. when starting up with the topology we can reach received right away, but we don't see an ack until

```
public void acknowledgeEpoch(EpochReady ready)
    {
        ready.metadata.addCallback(() -> {
            synchronized (AbstractConfigurationService.this)
            {
                epochs.acknowledge(ready);
            }
        });
```

so `accord.local.CommandStore#bootstrapper` is the main thing blocking the metadata... so it would be possible that a new node joins the ring, has to bootstrap its state, then receive topology updates.... so waiting on `minEpoch` still makes sense to me...

Reply:
Do you have the SHA that caused this? It doesn't quite track to me what we're doing here. If the burn test wasn't ACKing epoch 1, I don't see how this change would help either, so would be great to debug an example so can understand the cause/fix.

Reply:
I don't remember a sha/seed, would have to splunk through slack to see if I could find that.  It happened very often with this patch without this change though, not sure if other changes made it less helpful or not though.

So, the history was the following

```
T1: burn test starts with epoch=1, but does not setup the ack; this was a difference with C* which would ack
T2: epoch=2, but ack is slow due to timeouts/failures
T3: epoch=3, ack is slightly faster than epoch=2, so it wins the race and fails
```

I made a change to `start` so we ack the first topology, so if you call start (which we don't in C*) then `lastAcked !=0` and you wouldn't hit this issue.  In the C* case we stored the state locally and on reboot we delay ack until we can replay the events again, so it could be possible to hit this there if the oldest topology had not seen all replies yet.

Reply:
>  In the C* case we stored the state locally and on reboot we delay ack until we can replay the events again

Correction, the very first topology we load from disk we ack

So yeah, with the change to `start` this change isn't 100% needed, mostly left just in case